<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionLeo"/>
    <addaction name="actionHarout"/>
    <addaction name="actionLea"/>
    <addaction name="actionInna"/>
    <addaction name="actionOluwasayo"/>
    <addaction name="actionKodjo"/>
    <addaction name="actionJonas"/>
    <addaction name="actionYoucef"/>
    <addaction name="actionMaroua"/>
    <addaction name="actionFlorian"/>
    <addaction name="actionKarim"/>
    <addaction name="actionHugo"/>
    <addaction name="actionHenri"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLeo">
   <property name="text">
    <string>Léo</string>
   </property>
  </action>
  <action name="actionHarout">
   <property name="text">
    <string>Harout</string>
   </property>
  </action>
  <action name="actionLea">
   <property name="text">
    <string>Léa</string>
   </property>
  </action>
  <action name="actionInna">
   <property name="text">
    <string>Inna</string>
   </property>
  </action>
  <action name="actionOluwasayo">
   <property name="text">
    <string>Oluwasayo</string>
   </property>
  </action>
  <action name="actionKodjo">
   <property name="text">
    <string>Kodjo</string>
   </property>
  </action>
  <action name="actionJonas">
   <property name="text">
    <string>Jonas</string>
   </property>
  </action>
  <action name="actionYoucef">
   <property name="text">
    <string>Youcef</string>
   </property>
  </action>
  <action name="actionMaroua">
   <property name="text">
    <string>Maroua</string>
   </property>
  </action>
  <action name="actionFlorian">
   <property name="text">
    <string>Florian</string>
   </property>
  </action>
  <action name="actionKarim">
   <property name="text">
    <string>Karim</string>
   </property>
  </action>
  <action name="actionHugo">
   <property name="text">
    <string>Hugo</string>
   </property>
  </action>
  <action name="actionHenri">
   <property name="text">
    <string>Henri</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
